VEHICLE/SetRandomTrains.md

```diff
@@ -8,6 +8,36 @@ ns: VEHICLE
 void SET_RANDOM_TRAINS(BOOL toggle);
 ```
 
+**Information**
+Enables spawning random trains on the preset tracks. However to be able to use this call you would first need to call at least two other calls to make it function properly.
+
+You would need to tell the game which tracks you want to be enabled (or disabled), and you would need to set the (random) spawn frequency. If you don't call at least those to functions this function won't do anything at all!
+
+**Required other functions:**
+SwitchTrainTrack(TrackId: Number, Active: Boolean): [SWITCH_TRAIN_TRACK](#_0xFD813BB7DB977F20)
+SetTrainTrackSpawnFrequency(TrackId: Number, Intervalms: number): [SET_TRAIN_TRACK_SPAWN_FREQUENCY](#_0x21973BBF8D17EDFA)
+
+**Example (Lua)**
+The example snippet bellow will activate the 'large train tracks' around LS and into Sandy Shores etc, and the Metro in Los Santos.
```

Comment:
Would probably be better with including line 21 as a comment in the codeblocks.

Reply:
Thanks, was one of my first contribs will check into it and into the contrib guidelines

Reply:
I suppose this is more like expected and according the guidelines?
If it's reviewed as being correct i will also adapt the other pull request to this

Reply:
It should just be `## Examples`